src/components/AnnouncementInfo/AnnouncementInfo.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react'
+import { DocumentRenderer } from '@keystone-6/document-renderer'
+import { InferRenderersForComponentBlocks } from '@keystone-6/fields-document/component-blocks'
+import styles from './AnnouncementInfo.module.scss'
+import { componentBlocks } from 'components/ComponentBlocks/component-blocks'
+import { AnnouncementRecord } from 'types'
+import LinkTo from 'components/util/LinkTo/LinkTo'
+
+const AnnouncementInfo = ({
+  announcement,
+}: {
+  announcement: AnnouncementRecord
+}) => {
+  const {
+    title,
+    publishedDate,
+    body: { document },
+  } = announcement
+
+  const componentBlockRenderers: InferRenderersForComponentBlocks<
+    typeof componentBlocks
+  > = {
+    callToAction: (props: any) => {
+      const { ctaText, link } = props
+      // const {
+      //   link: { value },
+      // }: HydratedRelationshipData = props
```

Comment:
question: Do we need to leave the comment here? Also I'm new to TypeScript but looks like the type `HydratedRelationshipData` was dropped, wondering why it's not needed? Not requesting a change just trying to understand.

Reply:
`HydratedRelationshipData` is provided by Keystone and I thought that it would infer all of the types for me, but just kept giving me type errors. I used type `any` for props just to test if the feature would work, but forgot to loop back and create a custom type for the props. Thanks for pointing this out!

Reply:
Update on this: I removed the comment, but left the type as `any` for now. Keystone docs basically tell you to rely on autocomplete in your IDE for configuring types, but after messing with it for a bit, I'm not sure what it wants. Might need to pair on this later.